features/support/helpers/form_helper.rb

```diff
@@ -8,9 +8,9 @@ def submit_form_with(table)
 
   def fill_form_with(table)
     table.rows_hash.each do |name, value|
-      if has_select?(name, wait: 0)
+      if has_select?(name)
```

Comment:
wait till the element appears (if it's already there - there is no penalization)